x-pack/plugins/fleet/server/services/epm/packages/remove.ts

```diff
@@ -24,6 +24,7 @@ import { packagePolicyService, appContextService } from '../..';
 import { splitPkgKey } from '../registry';
 import { deletePackageCache } from '../archive';
 import { removeArchiveEntries } from '../archive/save_to_es';
+import { deleteIlms } from '../elasticsearch/datastream_ilm/remove';
```

Comment:
same here, is this file missing from the commit?

Reply:
yes, updated